Modules/Microsoft365DSC/DSCResources/MSFT_EXOTransportConfig/MSFT_EXOTransportConfig.schema.mof

```diff
@@ -0,0 +1,36 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXOTransportConfig")]
+class MSFT_EXOTransportConfig : OMI_BaseResource
+{
+    [Write, Description("The AddressBookPolicyRoutingEnabled parameter controls how recipients are resolved in an organization that uses address book policies to create separate virtual organizations within the same Exchange organization.")] Boolean AddressBookPolicyRoutingEnabled;
```

Comment:
None of the parameter has been configured as Key parameter. Does this resource work properly in the current state?

If the resource can only be used once, for example because you configure the settings tenant wide, please use the IsSingleInstance option like is done in the SPOTenantSettings: https://github.com/microsoft/Microsoft365DSC/blob/a64cfc6f94cb7366aaadc1bc499f83080de30be2/Modules/Microsoft365DSC/DSCResources/MSFT_SPOTenantSettings/MSFT_SPOTenantSettings.schema.mof#L4

Reply:
Added IsSingleInstance parameter